i18n/po/i18n_nl.po

```diff
@@ -49,6 +64,21 @@ msgstr ""
 msgid "Merged schema {0} into schema {1}"
 msgstr ""
 
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:7
+#, java-format
+msgid "Message at {0} ({1}):  {2}"
+msgstr "Bericht bij {0} ({1}): {2}"
+
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:11
+#, java-format
+msgid "Multiple values for Primitive Object constraint {0}: {1}"
+msgstr "Meerere waarden voor Primitive Object constraint {0}: {1}"
```

Comment:
Meer**d**ere waarden.

Also not so sure if this message should be like this :)

Reply:
I have fixed the typo. The other comment applies also to the English message, so it is out of scope of this pull request.